src/activity-functions-function-notation.xml

```diff
@@ -160,24 +161,22 @@
         <blockquote>
             <p><m>f(2) = 30</m></p>
         </blockquote>
-
-        <p>Let's piece apart the English sentence so you can see the structure as it compares to math. In English we tell the reader what we are talking about. This is the subject, <q>The position of the train <m>2</m> hours after 8 <init>a.m.</init></q> and in math we can simply say, <m>f(2)</m>.</p>
-
-        <p>Now the reader is wondering, so what about this train's position?  Is it more than something, less than something, is growing, shrinking, what? In our example we just want to say what the distance is. In math <q>is</q> means <q>equals</q> but we use the symbol, <m>=</m>.</p>
-
-        <p>Finally we want to tell the reader the punch line.  We want to tell them that the distance is <m>30</m> miles from downtown at the specified time. Hence, <m>f(2) = 30</m> is the entire senctence.</p>
     </subsection>
 
     <subsection>
         <title>The Grammar of Function Notation</title>
 
-        <p>The math <q>sentence</q> above is actually an equation and it uses <term>function notation</term><index><main>function</main><sub>notation</sub></index><index><main>notation</main><sub>for functions</sub></index> to express the information about the train. Of course, the equation is meaningless unless the reader knows how the variables and the function is defined.  If you know Spanish or <q>math</q> you can <em>interpret</em> the meaning of something you read in those languages into English.  Likewise, if you know <q>math language</q> you can interpret function notation its meaning into English too. There will be much interpretation in this class.</p>
+        <p>The math <q>sentence</q> above is actually an equation and it uses <term>function notation</term><index><main>function</main><sub>notation</sub></index><index><main>notation</main><sub>for functions</sub></index> to express the information about the train's position at a particular time.</p>
+
+        <p>In math we don't really have subjects or verbs or objects. Instead we have input values (numbers) and we have output values. The function is the <q>machine</q> or the rule that tells us how to use the input to make an output.</p>
 
-        <p>In math we use variables and symbols to replace entire words or even entire sentences. Instead of <q>Train's distance in miles from downtown</q> we can use the variable <m>D</m>. Instead of the <q>number of hours after 8 a.m.</q>, we can use the variable <m>t</m>.</p>
+        <p>Variables are used to represent values we don't know or that may change. The distance from downtown is constantly changing with time, so we can let <m>D</m> represent the <q>Train's distance in miles from downtown</q>.</p>
+        <p>For the <q>number of hours after 8 a.m.</q>, we can use the variable <m>t</m> where t counts how many hours have passed since 8 <init>a.m.</init></p>
```

Comment:
Only use `<init>` with "a.m." when that last period in "a.m." is going to be followed by a space and is _not_ going to end a sentence. One "a.m." here is followed by a comma, and the other is at the end of a sentence. So neither instance needs `<init>`. (`<init>` exists only because LaTeX assumes a period followed by a space represents an end-of-sentence, and so puts extra space there. And `<init>` prevents the extra space. So you really do not want it at an actual end of sentence or it will kill space that should actually be there.)